deployments/porch/2-function-runner.yaml

```diff
@@ -37,16 +37,15 @@ spec:
       serviceAccountName: porch-fn-runner
       containers:
         - name: function-runner
-          image: gcr.io/example-google-project-id/porch-function-runner:latest
+          image: nephio-project/porch-function-runner:latest
```

Comment:
Not sure if this might cause issues with some of the other Make targets.

Reply:
Seems like `scripts/create-deployment-blueprint.sh` is looking for those, for replacing them. Fixed it, but I couldn't get the associated make target working, seems like has some google things in there.

Reply:
Ye I think some/most of that google specific stuff should be removed.
I started using the same pkg when doing the kind deploy target but switched to use the catalog pkg instead.
https://github.com/nephio-project/porch/blob/main/Makefile#L325

We need to refine the Make file to reduce the bloat.